packages/components/src/StoryFeed/StoryFeed.stories.tsx

```diff
@@ -16,6 +16,31 @@ const contributors = [
   { avatar: "https://picsum.photos/50", username: "ronburgundy", amount: "0.009214 ETH" },
 ];
 
+const newsroom = {
+  contractAddress: "0x224b07d2ff8ca3c3757fdfc101922fb4792cc2d7",
+  multisigAddress: "0xe1e345504d9cd4d19bee9c0300a8c8265e62b17c",
+  charter: {
+    name: "Coda Story",
+    newsroomUrl: "https://codastory.com",
+    mission: {
+      purpose:
+        "Coda Story puts a team of journalists on one crisis at a time and stays with it, providing unique depth, continuity and understanding to events that shape our world.",
+    },
+  },
+};
+
+const openGraphData = {
+  article: {
+    published_time: "2019-08-28T15:58:32Z",
+  },
+  description: "Uyghurs are gaming TikTok’s algorithm to find a loophole in Xinjiangs’s information lockdown",
+  images: {
+    url: "https://codastory.com/wp-content/uploads/2019/09/Still-Header-.png",
```

Comment:
if this were to match how it's stored on the backend, images is actually an array of objects. so accessing the image would be images[0].url